static/css/style.css

```diff
@@ -13,4 +13,6 @@ body {
     top: 0;
     bottom: 0;
     width: 100%;
+    height: 100%;
+    background-color: #333333;
```

Comment:
When you're zooming around you'll see the background before the tiles load. I found that a darker color was much nicer to look at than bright white. That was in the middle of the night though :) so we can scratch it if you like.